### PR TITLE
chore(liveness): add error handling for no chunks being recorded

### PR DESCRIPTION
**Pull request by thaddmt**
#### Description of changes
- Added error handling for case where no chunks are recorded for a full session, the component should throw an error in this case

Screenshot 2023-06-27 at 2 49 50 PM

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/react-liveness/src/components/FaceLivenessDetector/service/utils/videoRecorder.ts [hunk 1]
```diff
@@ -64,6 +64,10 @@ export class VideoRecorder {
     this._recorder.dispatchEvent(event);
   }
 
+  getVideoChunkSize(): number {
+    return this._chunks.length;
```
Comment (calebpollman) [inline packages/react-liveness/src/components/FaceLivenessDetector/service/utils/videoRecorder.ts:68]: Any chance of `this._chunks` being falsy?

File: packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts [hunk 2]
```diff
@@ -740,7 +740,9 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
       },
       callErrorCallback: async (context, event) => {
         const errorMessage =
-          event.data?.error?.message || event.data?.error?.Message;
+          event.data?.error?.message ||
+          event.data?.error?.Message ||
+          event.data?.message;
```
Comment (calebpollman) [inline packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts:745]: A bit worried abt the deviation in the error related shapes that `event.data` has here, is there potential that we may be missing any and/or there a way that the shape can be standardized?

Comment (thaddmt) [reply packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts:745]: Yeah we're planning to do more standardization of this in this other pr with a breaking change - https://github.com/aws-amplify/amplify-ui/pull/4134

Issue being that with events so far when we have created custom events we have used the pattern `{ data: { error } }` but by default when xstate handles error events they use `{ data: new Error }`

**Review by calebpollman — APPROVED** [review]

**Review by ioanabrooks — APPROVED** [review]

File: packages/react-liveness/src/components/FaceLivenessDetector/service/utils/videoRecorder.ts (hunk 1 above)
Comment (thaddmt) [reply packages/react-liveness/src/components/FaceLivenessDetector/service/utils/videoRecorder.ts:68]: shouldn't be, we initialize it to empty array

**Pull request merged**